packages/hyperion-autologging/src/ALSurface.ts

```diff
@@ -40,7 +40,7 @@ export type DataType = ALFlowletDataType;
 export type FlowletType = Flowlet<DataType>;
 export type FlowletManagerType = ALFlowletManager<ALFlowletDataType>;
 type ALChannelEventType = ALChannelSurfaceEvent;
-type ALChannel = Channel<ALChannelEventType>;
+export type ALChannel = Channel<ALChannelEventType>;
```

Comment:
In this repo, I had each module define what structure it expects from the Channel. That way it is more composable. 

At the top of the `ALUIEventPublisher, there should be a code that defines the channel it expects.